public/app/screens/main/navigation/navigation.jsx

```diff
@@ -27,7 +27,7 @@ var Navigation = React.createClass({
     }.bind(this));
 
     return (
-      <div className={'cc_navigation ' + this.props.orientation}>
+      <div className={'cc_navigation icon-bar two-up' + ' ' + this.props.orientation}>
```

Comment:
Nor sure about that. 
TODO is smth which need attention soon, not the generic comment.

But will add the comment, though.